static_src/util/cf_api.js

```diff
@@ -423,6 +423,34 @@ export default {
   },
 
   // TODO refactor with org user permissions
+  postCreateNewUserWithGuid(userGuid) {
+    // return http.put(`${APIV}/users/${userGuid}/${role}/${orgGuid}/`)
+    return http.post(`${APIV}/users`, {
+      guid: userGuid
+    })
+      .then(res => this.formatSplitResponse(res.data))
+      .catch(res => {
+        if (res && res.response && res.response.status === 400) {
+          // The user is unauthenicated.
+          return Promise.resolve({ guid: userGuid });
+        }
+        // At this point we're not sure if the user is auth'd or not. Treat it
+        // as an error condition.
+        const err = parseError(res);
+
+        // Let someone else handle the error
```

Comment:
Tried to change this, but the tests were failing because the promise return wasn't explicit. Re-added it.